SUSTechTISHelper.js

```diff
@@ -472,8 +443,71 @@ async function fetchPointFromAPI() {
     }
 }
 
+function checkPointUpdate() {
+
+    if (need_update) {
```

Comment:
`!need_update`?

Reply:
If `need_update` is true then we simply wait for the `fetchPointFromAPI` function to execute. 
The opposite if-statement is at the beginning of `fetchPointFromAPI` function. This makes sure only one of the two functions will be executed at the same time. 
I do this because function invoked by `setInterval` will not block main thread but run asynchronously, which means `checkPointUpdate` and `fetchPointFromAPI` will start simultaneously (because they are both invoked by `setInterval` in `startReferesh`). I don't wish both of them read and write to `need_update`, as this might cause syncronous issue.

Reply:
We won't have syncronous issue here because js is single threaded.